2 - RTB/Documentazione esterna/Piano di Qualifica/Piano di Qualifica.typ

```diff
@@ -0,0 +1,34 @@
+#import "/template.typ": *
+
+#show: project.with(
+  title:"Piano di Qualifica",
+  subTitle: "",
+  authors: (
+    "Todesco Mattia",
+  ),
+  reviewers:(
+    "Nardo Silvio",
+  ),
+
+  showIndex: true,
+  showLog: true,
+  isExternalUse: true,
+);
+
+= Introduzione
+
+== Scopo del documento
+Il presente documento viene redatto con lo scopo di definire gli standard di qualità e di valutazione del prodotto finale. Essi saranno definiti conformemente ai requisiti e a ciò che il Proponente richiede.
```

Comment:
- in teoria il documento serve a definire le procedure con cui valutiamo la qualità del progetto soprattutto in fase di sviluppo e, per come è la frase iniziale, mi sembra che il documento serva solo ad analizzare il prodotto finale.
- metterei "Essi saranno definiti conformemente ai requisiti e alle richieste del Proponente."